portfolio/src/main/webapp/index.html

```diff
@@ -207,44 +207,21 @@ <h4>my relevant coursework</h4>
 
         <section id="comments">
           <h3>arrington's friends comments</h3>
-          <p>displaying <span class="num">4</span> of <span class="num">12345</span> comments</p>
+          <form action="/data?display=10" method="POST">
+                displaying 
+                <select name="num-comments" id="num-comments" onchange="this.form.submit()">
+                <option>10</option>
+                <option>25</option>
+                <option>50</option></select> comments.
+          </form>
```

Comment:
It might be good to use a different DataServlet for this request so that it has a distinct url. Determining if a post should update the number of comments shown based on whether or not `?display=10` is present seems a bit fragile.

Why is `?display=10` in the url when `num-comments` is the one being used to actually set the count?
Maybe the action should just be something like `action="/settings"`

Reply:
This was just something they told us to add for testing and I forgot to remove it. I've removed the `?display-num` and it still works